direct/data/transforms.py

```diff
@@ -16,6 +16,44 @@
 from direct.utils.asserts import assert_complex, assert_same_shape
 
 
+class FFT(torch.autograd.Function):
+    @staticmethod
+    def symbolic(g, data, dim, centered, normalized, inverse=False):
+        return g.op("IFFT" if inverse else "FFT", data, centered_i=int(centered), dim_i=list(dim))
+
+    @staticmethod
+    def forward(self, data, dim, centered, normalized, inverse=False):
+        if not all((_ >= 0 and isinstance(_, int)) for _ in dim):
```

Comment:
Please rename `fft2` to `origin_fft2` and call `origin_fft2` inside FFT's forward. It's needed to reduce code changes